sw/airborne/modules/guidance/gvf/gvf.h

```diff
@@ -53,6 +53,14 @@ typedef struct {
   float p[16];
 } gvf_tra;
 
+typedef struct {
+  int seg;
```

Comment:
can you please add some inline doxygen comments on what these variables mean and what unit they have?